spec/system/candidate_interface/navigation_tabs_spec.rb

```diff
@@ -20,6 +21,12 @@
     when_i_click_on_your_applications
     then_i_see_your_applications_as_active
 
+    when_i_click_on_an_application
+    then_i_see_your_applications_as_active
+
+    when_i_click_on_change_course
+    then_i_see_your_applications_as_active
+
```

Comment:
I've only added a couple of extra tests here - not sure how far to go with it. 
Should we add tests for every page 🤷‍♂️

Reply:
I think the unit tests on the `ChoicesControllerMatcher` are enough. I'm not pushed to cover every scenario in system tests, personally.